apps/web/src/actions/evaluations/generateSuggestedEvaluations.ts

```diff
@@ -0,0 +1,41 @@
+'use server'
+
+import { ChainObjectResponse } from '@latitude-data/core/browser'
+import { findAllEvaluationTemplates } from '@latitude-data/core/data-access'
+import { Latitude } from '@latitude-data/sdk'
+import { SuggestedEvaluation } from '$/stores/suggestedEvaluations'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const generateSuggestedEvaluationsAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      documentContent: z.string(),
+    }),
+  )
+  .handler(async ({ input }) => {
+    const templates = await findAllEvaluationTemplates().then((r) => r.unwrap())
+    const templateString = templates
+      .map((t) => `${t.id}\n${t.name}\n${t.description}\n`)
+      .join('\n')
+    const sdk = new Latitude('d7c94cc1-72a8-4252-b1a8-3e6b3cf9c29f', {
```

Comment:
👀

Reply:
yes yes i have to move all these to env vars (this api key is local no worries i'm not leaking anything)